### PR TITLE
Localization bugfix & improvement

### PR DESCRIPTION
**Pull request by Antifluxfield**
https://github.com/Blood-Asp/GT5-Unofficial/pull/1346

### CODE REVIEW

File: src/main/java/gregtech/GT_Mod.java [hunk 1]
```diff
@@ -294,6 +294,7 @@ public void onPreLoad(FMLPreInitializationEvent aEvent) {
         gregtechproxy.gt6Pipe = tMainConfig.get("general", "GT6StyledPipesConnection", true).getBoolean(true);
         gregtechproxy.gt6Cable = tMainConfig.get("general", "GT6StyledWiresConnection", true).getBoolean(true);
         gregtechproxy.costlyCableConnection = tMainConfig.get("general", "CableConnectionRequiresSolderingMaterial", false).getBoolean(false);
+        GT_LanguageManager.i18nPlaceholder = gregtechproxy.costlyCableConnection = tMainConfig.get("general", "EnablePlaceholderForMaterialNamesInLangFile", true).getBoolean(true);
```
Comment (Technus) [inline src/main/java/gregtech/GT_Mod.java]: Why does it duplicate the line above?

Comment (Technus) [reply src/main/java/gregtech/GT_Mod.java]: Actually why does it assign something to costly cable connection.

Comment (Dream-Master) [reply src/main/java/gregtech/GT_Mod.java]: @Technus merge or did this need to changed first ?

Comment (Antifluxfield) [reply src/main/java/gregtech/GT_Mod.java]: Typo

Comment (Technus) [reply src/main/java/gregtech/GT_Mod.java]: If u can push a commit that fixes it.

Comment (Dream-Master) [reply src/main/java/gregtech/GT_Mod.java]: @Technus done

**Comment by Antifluxfield** [comment]
Just failed to catch that mistake... Fixed

**Comment by Dream-Master** [comment]
@Antifluxfield thanks

**Pull request merged**